TextAnalyzer/TemplateCommandOption.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TextAnalyzer
+{
+    class TemplateCommandOption : ICommand
+    {
+        public string Description
```

Comment:
public string Description => "Template option, writes the text file in the console";

Reply:
Adjusted 👍